prow/gerrit/client/client.go

```diff
@@ -51,9 +51,12 @@ const (
 	// New status indicates a Gerrit change is new (ie pending)
 	New = "NEW"
 
-	// ReadyForReviewMessage is the message for a Gerrit change if it's changed
-	// from Draft to Active
-	ReadyForReviewMessage = "Set Ready For Review"
+	// ReadyForReviewMessage are the messages for a Gerrit change if it's changed
+	// from Draft to Active.
+	// This message will be sent if users press the `MARK AS ACTIVE` button.
+	ReadyForReviewMessage1 = "Set Ready For Review"
```

Comment:
nit: not very descriptive var names, how about `ReadyForReviewMessageFixed` and `ReadyForReviewMessageCustomizable`?

Reply:
Done.